### ISSUE TITLE
Improve visual styles on drop down menu titles 

### ISSUE THREAD

**Issue by markbjadu** [jadu/pulsar#931]
The title colour contrast in the drop down should be improved to make it easier to read on first glance it's difficult to find exactly what the user is looking for.

1. Create a modern form
2. Click on 'Templates'
3. Create a template
4. See the drop down menu

The titles in the drop down menu should be clear.

 - macOS Mojave 10.14.2  
 - Browser chrome latest
 - Version 71.0.3578.98 (Official Build) (64-bit)

This might be a problem on other browsers and windows IE as well.

![screenshot_2018-12-19_at_10_17_42](https://user-images.githubusercontent.com/6906173/50214803-10d79200-0379-11e9-8da1-1a8f5553d80a.png)

**Comment by Stanton** [comment]
Discussed with Mark over Slack, we're aware that this is default operating system styling but we will investigate to see if we can make any improvements to the colour contrast of optgroup labels.

**Comment by markbjadu** [comment]
![screenshot 2018-12-19 at 11 13 15](https://user-images.githubusercontent.com/6906173/50216943-264fba80-037f-11e9-982c-908c7be17c1f.png)
![screenshot 2018-12-19 at 11 13 07](https://user-images.githubusercontent.com/6906173/50216945-264fba80-037f-11e9-898d-31da9b6beece.png)

Here is what it looks like on IE11 on Windows. This is what I would expect it to look like.

**Comment by Stanton** [comment]
@markbjadu The styling you reported in Chrome is the default operating system styling, while I agree the contrast of optgroup labels isn't great, we may decide not to change this as it's a system-level control.

**Comment by Stanton** [comment]
Dark mode
![screenshot 2019-02-13 at 16 05 08](https://user-images.githubusercontent.com/18653/52725460-2b10a380-2fa9-11e9-95cf-3d96a9ee1acc.png)

Light mode
![screenshot 2019-02-13 at 16 05 47](https://user-images.githubusercontent.com/18653/52725496-3fed3700-2fa9-11e9-81e1-af29fb4b76b9.png)

Light mode + High contrast accessibility setting
![screenshot 2019-02-13 at 16 04 27](https://user-images.githubusercontent.com/18653/52725533-52677080-2fa9-11e9-9688-1dabc97a0997.png)

**Comment by Stanton** [comment]
After some reading and discussion with the Pulsar team we've decided not to resolve this issue.

The select elements are styled by the operating system, contrast can be increased by using OS accessibility settings, this fact, coupled with the difficulty in styling optgroup labels consistently across browsers means we will not seek to resolve this issue within Pulsar.
